### PR TITLE
Update vaccination consent screen (DEV)

### PR DESCRIPTION
**Pull request by jurajkusnier**
- Update layout and string in:
  - vaccination consent screen
  - vaccination registration home screen card
### Testing 
- switch phone language to `DE` to see updated strings

### CODE REVIEW

File: Corona-Warn-App/src/main/res/layout/trace_location_privacy_card.xml [hunk 1]
```diff
@@ -36,11 +36,11 @@
         layout="@layout/include_bullet_point"
         android:layout_width="@dimen/bullet_point_size"
         android:layout_height="@dimen/bullet_point_size"
-        app:layout_constraintBaseline_toBaselineOf="@id/first_bulletpoint_title"
```
Comment (axelherbstreith) [inline Corona-Warn-App/src/main/res/layout/trace_location_privacy_card.xml:39]: Is the renaming in this file intended? Or just a side effect from the refactoring of `vaccination_consent_privacy_card.xml`

File: Corona-Warn-App/src/main/res/layout/vaccination_consent_fragment.xml [hunk 2]
```diff
@@ -122,7 +122,7 @@
                 android:importantForAccessibility="no"
                 app:layout_constraintStart_toStartOf="@id/vaccination_consent_qr_code_icon"
                 app:layout_constraintTop_toBottomOf="@id/vaccination_consent_qr_code_text"
-                app:srcCompat="@drawable/ic_qr_time" />
+                app:srcCompat="@drawable/ic_shield_illustration" />
```
Comment (axelherbstreith) [inline Corona-Warn-App/src/main/res/layout/vaccination_consent_fragment.xml:126]: `marginVertical` for `MoreInformationView` seems a bit to large in the layout when comparing it to figma

File: Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml [hunk 3]
```diff
@@ -58,11 +58,11 @@
         Homescreen cards
     ###################################### -->
     <!-- XHED: Title for Vaccination Certificate Registration Home Card -->
-    <string name="vaccination_card_registration_title">"Impfzertifikat registrieren"</string>
+    <string name="vaccination_card_registration_title">"Impfzertifikat hinzufügen"</string>
     <!-- YTXT: Body text for Vaccination Certificate Registration Home Card -->
-    <string name="vaccination_card_registration_body">"Scannen Sie den QR-Code auf Ihrem Impfzertifikat, um Ihre Impfung zu registrieren und einen digitalen Impfnachweis zu erhalten."</string>
+    <string name="vaccination_card_registration_body">"Fügen Sie Ihre Impfzertifikate in die App hinzu, um sie immer sich zu haben. Scannen Sie dafür den QR-Code Ihres Dokuments."</string>
```
Comment (vaubaehn) [inline Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml:63]: ... im sie immer __bei__ sich zu haben

**Review by vaubaehn — COMMENTED** [review]
One word missing in strings

File: Corona-Warn-App/src/main/res/layout/trace_location_privacy_card.xml (hunk 1 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/layout/trace_location_privacy_card.xml]: yeah, refactoring :(

File: Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml (hunk 3 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml]: thanks, good catch

File: Corona-Warn-App/src/main/res/layout/vaccination_consent_fragment.xml (hunk 2 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/layout/vaccination_consent_fragment.xml:126]: updated

**Review by axelherbstreith — APPROVED** [review]
LGTM 👍 Tested on Samsung Galaxy S8

**Review by chiljamgossow — APPROVED** [review]
LGTM

**Review by ChristinaTsigganos — APPROVED** [review]
UA approved

**Pull request merged**